.github/workflows/main.yaml

```diff
@@ -0,0 +1,63 @@
+name: Main branch CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+  schedule:
+    - cron: '0 6 * * 0'
+
+jobs:
+  check-libraries:
+    uses: canonical/sdcore-github-workflows/.github/workflows/check-libraries.yaml@v0.0.1
+    secrets:
+      CHARMCRAFT_AUTH: ${{ secrets.CHARMCRAFT_AUTH }}
+
+  lint-report:
+    uses: canonical/sdcore-github-workflows/.github/workflows/lint-report.yaml@v0.0.1
+
+  terraform-check:
+    uses: canonical/sdcore-github-workflows/.github/workflows/terraform.yaml@v0.0.1
+
+  static-analysis:
+    uses: canonical/sdcore-github-workflows/.github/workflows/static-analysis.yaml@v0.0.1
+
+  unit-tests-with-coverage:
+    uses: canonical/sdcore-github-workflows/.github/workflows/unit-test.yaml@v0.0.1
+
+  integration-test:
+    uses: canonical/sdcore-github-workflows/.github/workflows/integration-test.yaml@v0.0.1
+    with:
+      charm-file-name: "sdcore-gnb-integrator_ubuntu-22.04-amd64.charm"
+
+  publish-charm:
+    name: Publish Charm
+    needs:
+      - lint-report
+      - static-analysis
+      - unit-tests-with-coverage
+      - integration-test
+    if: ${{ github.ref_name == 'main' }}
+    uses: canonical/sdcore-github-workflows/.github/workflows/publish-charm.yaml@v0.0.1
+    with:
+      charm-file-name: "sdcore-gnb-integrator_ubuntu-22.04-amd64.charm"
+      track-name: "1.0"
```

Comment:
Do we want to align the versioning with other charms? One part of me says "no, because it's not part of the Core", but then the other part thinks "it's the easiest way to provide information about compatibility with particular release of Charmed Aether SD-Core" ;)

Reply:
1.4 it is!